src/constants/notification-content.js

```diff
@@ -153,13 +153,18 @@ export default function getNotificationContent({ type, payload }) {
 
     case "manual_transaction": {
       fragments = [
-        'The administrator',
+        'The administrator ',
       ];
       if (payload.transaction.change > 0) {
         fragments.push(`granted ${payload.transaction.change} innopoints to you`);
       } else {
         fragments.push(`deducted ${-payload.transaction.change} innopoints from you`);
       }
+      break;
+    }
+
+    default: {
+      fragments = ['Just a message from heaven wishing you a great day'];
```

Comment:
I was thinking more like "An error occurred, please report it", but that looks nice :joy: